pynta/calculator.py

```diff
@@ -285,7 +285,7 @@ def run_harmonically_forced_xtb_no_pbc(atoms,atom_bond_potentials,site_bond_pote
     hfxtb = HarmonicallyForcedXTB(method="GFN1-xTB",
                                   atom_bond_potentials=new_atom_bond_potentials,
                                  site_bond_potentials=new_site_potentials)
-
+    bigad.set_constraint(out_constraints)
```

Comment:
Should we also get rid of the `atoms.set_constraint(out_constraints)`?

Reply:
I left it but I don't think we need it.

Reply:
I added in a second commit to the branch that deletes that line after merging, so we should be good.